c2corg_api/views/waypoint.py

```diff
@@ -28,7 +29,11 @@
 from functools import lru_cache
 from sqlalchemy.orm import joinedload, load_only
 from sqlalchemy.orm.util import aliased
-from sqlalchemy.sql.expression import and_
+from sqlalchemy.sql.elements import literal_column
+from sqlalchemy.sql.expression import and_, union
+
+# the number of routes that are included for waypoints
+NUM_ROUTES = 30
```

Comment:
We'll have to argue about this limitation!
I have pointed the problem of v5 summits with dozens (or even hundreds of associated routes like  http://www.camptocamp.org/summits/107317/fr/massif-des-calanques) but the association seems relunctant to do anything...

At least would it be possible to raise a little this limit (for instance 50 or even 100)?